item_json_creator.py

```diff
@@ -0,0 +1,124 @@
+import json
+
+
+def parse_item_lines(item_lines):
```

Comment:
can you a comment on what the purpose of this file is or what method is first called to call all the others

Reply:
you simply run the file where there's a 'Rocket League Items.txt' and it creates the 'categorised_items.json' that the GUI uses to populate dropdowns.

Reply:
i meant in the code :P

Reply:
i realised as i scrolled down.

Reply:
Take note that in the future that file might not exist, i will make a function on the dll that automatically gets the items sorted.